src/test/java/SalesforceModules/QuoteUtil.java

```diff
@@ -1037,28 +1053,208 @@ public static void changeStatusAsDraft_ss(String status, String logStep) throws
         KeywordUtil.delay(3000);
         KeywordUtil.waitForClickable(ServiceSupportObject.editButton);
         WebElement order_status = KeywordUtil.getDriver().findElement(ServiceSupportObject.editButton);
-        JavascriptExecutor executor_draft = (JavascriptExecutor)KeywordUtil.getDriver();
+        JavascriptExecutor executor_draft = (JavascriptExecutor) KeywordUtil.getDriver();
         executor_draft.executeScript("arguments[0].click();", order_status);
 
         KeywordUtil.delay(8000);
         KeywordUtil.waitForVisible(ServiceSupportObject.selectStatus_ss);
-        KeywordUtil.click(ServiceSupportObject.selectStatus_ss,logStep);
+        KeywordUtil.click(ServiceSupportObject.selectStatus_ss, logStep);
         KeywordUtil.delay(3000);
         Thread.sleep(000);
-        String changeStatus_ss = "//lightning-base-combobox-item//span[@title='"+status+"']";
-        try{
+        String changeStatus_ss = "//lightning-base-combobox-item//span[@title='" + status + "']";
+        try {
             size = KeywordUtil.getDriver().findElements(By.xpath(changeStatus_ss)).size();
 
-        }catch (Exception e){}
+        } catch (Exception e) {
+        }
 
-        if(size==0){
+        if (size == 0) {
             KeywordUtil.takeScreenshotAndAttachInReport();
             System.out.println("no such status present");
             System.out.println(size);
-        }else {
+        } else {
             KeywordUtil.click(By.xpath(changeStatus_ss), "status changed as draft");
         }
 
         CasesUtil.clickSaveButton_ss("Saved");
     }
+
+    /**
+     * Create new quote combining all methods in billing section
+     *
+     * @param name,opportunityName the log
+     */
+    public static void createNewQuote_billing(String name, String opportunityName, String type) throws Exception {
+        clickQuoteButton_billing("Navigated to quote");
+        newButtonQuote_billing("Clicked new button");
+        checkPrimary("primary checkbox marked");
+        selectOpportunity_ss(opportunityName, opportunityName + " entered opportunities name");
+        selectAccount_billing(name, name + " entered account name");
+        selectType_ss(type, "selected type");
+        clickSaveButton("clicked save button");
+        addDiscountProduct_billing();
+    }
+
+
+    /**
+     * click Quote button for billing section
+     *
+     * @param logStep the log
+     */
+    public static void clickQuoteButton_billing(String logStep) throws InterruptedException {
+        KeywordUtil.delay(8000);
+        WebElement element = KeywordUtil.getDriver().findElement(QuoteObject.clickOnQuotes);
```

Comment:
Please create a method for javascript click and use it here

Reply:
done